lib/common.ml

```diff
@@ -2,10 +2,13 @@ open Json_encoding
 
 let pp_string fmt = Format.fprintf fmt "\"%s\""
 let pp_int fmt = Format.fprintf fmt "%d"
+let pp_bool fmt = Format.fprintf fmt "%b"
```

Comment:
Why not using `Format.pp_print_bool` directly ?

Reply:
For consistency with the existing `pp_int` above that could also use `Format.pp_print_int`, I suppose. Do you want me to use `Format.pp_print_bool` directly, or to define `let pp_bool = Format.pp_print_bool`?

Reply:
Yes, I'm not sure why they were defined in the first place. I would just use `Format.pp_print_*` directly everywhere.